valhalla/mjolnir/node_expander.h

```diff
@@ -70,6 +69,10 @@ struct Edge {
   // index of the target (end) node of the edge
   uint32_t targetnode_;
 
+  // to record the access of an edge
+  uint16_t fwd_access;
+  uint16_t rev_access;
```

Comment:
I removed the `driveableforward`  & `driveablereverse` booleans and replaced all logic depending on them with the access masks which are only set if there's no `Use::EmergencyAccess` and has access to a vehicular mode. 

I guess these would fit still into the `attributes`  bit field, now that there's 25 bits available, though then only 1 would be left (access has 12 bits in directed edges) and we'd need to re-write to this if we needed more attributes. So I'd leave it this way.